languages/payplug-fr_FR.po

```diff
@@ -703,6 +707,22 @@ msgstr "Votre boutique doit utiliser l'euro comme monnaie de paiement."
 msgid "your@email.com"
 msgstr "votre@email.com"
 
-#: src/Gateway/PayplugApi.php:133
-msgid "Your payment schedule simulation is temporarily unavailable. You will find this information at the payment stage."
-msgstr "La simulation de votre échéancier est temporairement indisponible. Vous retrouverez cette information à l'étape du paiement."
\ No newline at end of file
+msgid "More informations"
+msgstr "Plus d'informations"
+
+msgid "support.payplug.com/hc/en-gb"
+msgstr "support.payplug.com/hc/fr"
+
+#, fuzzy
```

Comment:
Hi @PPmmesquita , can you tell me why we have translation disable below ? Is it normal ?

Reply:
Hi @alopez-pp they are disabled because they don't exist in the .pot file, the .pot file is the file from where we generate all the translations, (via poedit), We import the pot file and sync it with the language files, that was why it is commented, I'll add this translations to the pot file so we can use this in a propper way.

Reply:
got it thanks !